### DISCUSSION TITLE
2.1.6.1

### DISCUSSION THREAD

**Discussion by yujitach** [yujitach/MenuMeters#267] [Announcements]
Fixes a bug in the menu of the memory meter which I introduced while refactoring. I am sorry for the mess.

<hr /><em>This discussion was created from the release <a href='https://github.com/yujitach/MenuMeters/releases/tag/2.1.6.1'>2.1.6.1</a>.</em>

**Comment by briantopping** [comment]
Thank you for your continued dedication to this project. It is an indispensable part of using a Mac for me and so many others. It is the first thing I always see is missing on an unfamiliar Mac. So many of us are grateful for your work and don't get a chance to say something often enough. Thank you!!

**Comment by Manetala** [comment]
блять как это закрыть теперь
